app/forms/questionnaire_step1_form.rb

```diff
@@ -0,0 +1,173 @@
+                :main_office_town,
+                :main_office_county,
+                :main_office_postcode,
+                :accept_customers_from,
+                :advice_in_person,
+                :advice_by_other_methods,
+                :free_initial_meeting,
+                :initial_meeting_duration,
+                :initial_advice_fee_structure,
+                :ongoing_advice_fee_structure,
+                :allow_customers_to_pay_for_advice,
+                :minimum_fixed_one_off_fee_amount
+
+  validates :firm_email_address,
+            presence: true,
+            length: {maximum: 50},
+            format: {with: /.+@.+\..+/}
+
+  validates :firm_telephone_number,
+            presence: true,
+            length: {maximum: 30},
+            format: {with: /\A[0-9 ]+\z/}
+
+  validates :main_office_line_1,
+            presence: true,
+            length: {maximum: 100}
+
+  validates :main_office_postcode,
+            presence: true,
+            format: {with: /\A[a-zA-Z\d]{1,4} [a-zA-Z\d]{1,3}\z/}
+
+  validates :main_office_line_2,
+            :main_office_town,
+            :main_office_county,
+            presence: true
+
+  validate :accept_customers_from_list_is_valid
+  validate :advice_in_person_list_is_valid
+  validate :advice_by_other_methods_list_is_valid
+  validate :free_initial_meeting_choice_is_valid
+  validate :initial_meeting_duration_choice_is_valid
+  validate :initial_advice_fee_structure_choice_is_valid
+  validate :ongoing_advice_fee_structure_choice_is_valid
+  validate :allow_customers_to_pay_for_advice_choice_is_valid
+
+  def accept_customers_from_options
+    @accept_customers_from_options ||=
+        I18n.t('questionnaire.step_one.section_four.regions').map {|item| item[:region] }
+  end
+
+  def advice_in_person_options
+    @advice_in_person_options ||=
+        I18n.t('questionnaire.step_one.section_five.regions').map {|item| item[:region] }
+  end
+
+  def advice_by_other_methods_options
+    @advice_by_other_methods_options ||=
+        I18n.t('questionnaire.step_one.section_five.advice_options').map {|item| item[:advice] }
+  end
+
+  def free_initial_meeting_options
+    @free_initial_meeting_options ||= ['Yes', 'No']
+  end
+
+  def initial_meeting_duration_options
+    @initial_meeting_duration_options ||= ['30 min', '60 min']
+  end
+
+  def initial_advice_fee_structure_options
+    @initial_advice_fee_structure_options ||=
+        I18n.t('questionnaire.step_one.section_seven.fee_options').map {|item| item[:fee] }
+  end
+
+  def ongoing_advice_fee_structure_options
+    @ongoing_advice_fee_structure_options ||=
+        I18n.t('questionnaire.step_one.section_seven.fee_options').map {|item| item[:fee] }
+  end
+
+  def allow_customers_to_pay_for_advice_options
+    @allow_customers_to_pay_for_advice_options ||= [
+        'From their own resources',
+        'From funds to be invested',
+        'Either'
+    ]
+  end
+
+  private
+
+  def accept_customers_from_list_is_valid
```

Comment:
@benlovell, @tomas-stefano I had originally tried this
```
validates :accept_customers_from,
          presence: true,
          inclusion: {
              in: ->(form) { form.accept_customers_from_options }
          }
```

but the problem with this is that this question allows multiple values, which I can't seem to get the inclusion validator to support. Any ideas? I can though change the other custom validations that only require a single value to this style, which is of course better.